### DISCUSSION TITLE
Two part recipes

### DISCUSSION THREAD

**Discussion by Phenix51** [nextcloud/cookbook#2591] [FAQ and user questions]
Is it currently possible to organize a recipe in two parts? Such as part 1 being the ingredients and instructions for a crust and the part two to list the ingredients and instructions for the filling? Or do I need to keep them as two separate recipes?  
Apologies, I don't see a way to do this without causing  issues with the ingredients section when trying to adjust the portions.

**Comment by christianlupus** [comment]
You can add a fake ingredient starting with `## `. With the instructions, this might work as well. Does this help you?
